src/main/java/quickcarpet/skyblock/mixin/FlowerPotBlockMixin.java

```diff
@@ -0,0 +1,95 @@
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import quickcarpet.skyblock.SkyBlockSettings;
+
+@Mixin(FlowerPotBlock.class)
+public abstract class FlowerPotBlockMixin extends Block {
+    
+    @Shadow
+    @Final
+    private static Map<Block, Block> CONTENT_TO_POTTED = Maps.newHashMap();
+    private Block content;
+    
+    protected FlowerPotBlockMixin(Block block_1, Settings block$Settings_1)
+    {
+        super(block$Settings_1);
+        this.content = block_1;
+        CONTENT_TO_POTTED.put(block_1, this);
+    }
+    
+    /**
+     * @author PR0CESS
+     */
+   
+    @Inject(method = "activate", at = @At(value = "INVOKE", target = "Lnet/minecraft/block/FlowerPotBlock;activate(Lnet/minecraft/block/BlockState;Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/util/Hand;Lnet/minecraft/util/hit/BlockHitResult;)V"))
+    private void doActivate(BlockState blockState_1, World world_1, BlockPos blockPos_1, PlayerEntity playerEntity_1, Hand hand_1, BlockHitResult blockHitResult_1) {
+        if (flowerPotChunkLoading && !flowerPotChunkLoadingPowered) {
+            ItemStack itemStack_1 = playerEntity_1.getStackInHand(hand_1);
+            Item item_1 = itemStack_1.getItem();
+            Block block_1 = item_1 instanceof BlockItem ? (Block)CONTENT_TO_POTTED.getOrDefault(((BlockItem)item_1).getBlock(), Blocks.AIR) : Blocks.AIR;
+            boolean boolean_1 = block_1 == Blocks.AIR;
+            boolean boolean_2 = this.content == Blocks.AIR;
+            if (boolean_1 != boolean_2) {
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ(), boolean_2);
+                //This works, but will checking if its another chunk be faster?
+                world_1.setChunkForced(blockPos_1.getX()-1, blockPos_1.getZ(), boolean_2);
+                world_1.setChunkForced(blockPos_1.getX()+1, blockPos_1.getZ(), boolean_2);
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()+1, boolean_2);
+                world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ()-1, boolean_2);
+            }
+        }
+    }
+    @Inject(method = "onBreak", at = @At(value = "INVOKE",target = "Lnet/minecraft/block/FlowerPotBlock;onBreak(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/entity/player/PlayerEntity;)V"))
+    public void onOnBreak(World world_1, BlockPos blockPos_1, BlockState blockState_1, PlayerEntity playerEntity_1) {
+        if (flowerPotChunkLoading) {
+            world_1.setChunkForced(blockPos_1.getX(), blockPos_1.getZ(), false);
+            //This works, but will checking if its another chunk be faster?
+            world_1.setChunkForced(blockPos_1.getX()-1, blockPos_1.getZ(), boolean_2);
```

Comment:
`boolean_2` doesn't exist here. Did you even try compiling this?

Reply:
I have not tried compiling this. I have been able to compile carpetmod on my computer.